src/pc/common/BitArray.js

```diff
@@ -1,13 +1,13 @@
-const assert = require('assert')
+// const assert = require('assert')
```

Comment:
Shouldn't we remove it completely instead of just commenting ?

Reply:
I think these alerts have value, and ideally we would keep them but automatically remove them depending on environment.
Not sure exactly how to do that, but in the mean time I think it's useful to leave them commented so we have them here when we want to do it

Reply:
ok